CHANGELOG.rst

```diff
@@ -8,6 +8,80 @@
 
 .. towncrier release notes start
 
+Pytest 3.2.0 (2017-07-13)
+=========================
+
+Deprecations and Removals
+-------------------------
+
+- All old-style specific behavior in current classes in the pytest's API is
+  considered deprecated at this point and will be removed in a future release.
+  This affects Python 2 users only and in rare situations. (`#2147
+  <https://github.com/pytest-dev/pytest/issues/2147>`_)
+
+- A deprecation warning is now raised when using marks directly into parameters
```

Comment:
Yep, that's better.

We should also add an example to the docs or here if possible on how to update the code, as mentioned by @fschulze in the ML.